docs/quick-start.md

```diff
@@ -219,7 +218,7 @@ When a user logs in with the Netlify Identity widget, they will be directed to t
   }
 </script>
 ```
-*Note this script require ES6 and will not work on IE11. For legacy browser support, you'll need to use a transpiler like [Babel](https://babeljs.io/).
+Note: This script requires modern JavaScript and will not work on IE11. For legacy browser support, use function expressions (`function () {}`) in place of the arrow functions (`() => {}`), or use a transpiler like [Babel](https://babeljs.io/).
```

Comment:
I would change "This script" to "This example script" so that people realize that they can do it however they want, this is just an example that uses modern JS.